static/app/views/performance/http/domainStatusPageURLs.tsx

```diff
@@ -0,0 +1,7 @@
+export const DOMAIN_STATUS_PAGE_URLS = {
+  '*.github.com': 'https://www.githubstatus.com',
+  '*.slack.com': 'https://slack-status.com',
+  '*.atlassian.net': 'https://status.atlassian.com/api',
+  '*.googleapis.com': 'https://status.cloud.google.com',
+  '*.codecov.io': 'https://status.codecov.com',
```

Comment:
this would be a super cool open source library!

so we maintain this list and publish it for anyone to use, this would encourage companies/external contributors to also help keep it up to date 👀

while we are here we should also add sentry to this list 😅

Reply:
I volunteer to help out here given I'm a little more familiar with the library development side of things.

Reply:
@AbhiPrasad sounds great! Maybe there's some prior art on this (e.g., collab with https://github.com/ivbeg/awesome-status-pages)